packages/admin/admin-rte/src/core/Rte.tsx

```diff
@@ -148,201 +153,232 @@ export const styleMap = {
-        Object.entries(options.customInlineStyles).forEach(([name, { style }]) => {
-            customStyleMap[name] = style;
-        });
-    }
+export type RteClassKey = "root" | "disabled" | "editor";
 
-    return (
-        <div ref={editorWrapperRef} className={rootClasses.join(" ")}>
-            <Controls editorRef={editorRef} editorState={editorState} setEditorState={onChange} options={options} disabled={disabled} />
-            <div
-                className={classes.editor}
-                style={{ "--comet-admin-rte-min-height": `${props.minHeight === undefined ? 240 : props.minHeight}px` } as React.CSSProperties}
-            >
-                <DraftJsEditor
-                    ref={editorRef}
-                    editorState={editorState}
-                    onChange={decoratedOnChange}
-                    handleKeyCommand={handleKeyCommand}
-                    handleReturn={handleReturn}
-                    keyBindingFn={keyBindingFn}
-                    customStyleMap={customStyleMap}
-                    blockRenderMap={blockRenderMap}
-                    onCopy={(editor, event) => {
-                        onDraftEditorCopy(editor, event as React.ClipboardEvent<HTMLElement>);
-                    }}
-                    onCut={(editor, event) => {
-                        onDraftEditorCut(editor, event as React.ClipboardEvent<HTMLElement>);
-                    }}
-                    handlePastedText={(text: string, html: string | undefined, editorState: EditorState): DraftHandleValue => {
-                        const nextEditorState = pasteAndFilterText(html, editorState, options);
-
-                        if (nextEditorState) {
-                            decoratedOnChange(nextEditorState);
-                            return "handled";
-                        }
-
-                        return "not-handled";
-                    }}
-                    {...options.draftJsProps}
-                />
-            </div>
-        </div>
-    );
-};
+type OwnerState = Pick<RteProps, "disabled" | "minHeight">;
 
-export type RteClassKey = "root" | "disabled" | "editor";
+const Root = styled("div", {
+    name: "CometAdminRte",
+    slot: "root",
+    overridesResolver({ ownerState }: { ownerState: OwnerState }, styles) {
+        return [styles.root, ownerState.disabled && styles.disabled];
+    },
+})<{ ownerState: OwnerState }>(
+    ({ theme }) => css`
+        border: 1px solid ${getRteTheme(theme.components?.CometAdminRte?.defaultProps).colors.border};
+        border-top-width: 0;
+        background-color: #fff;
+    `,
+);
+
+const Editor = styled("div", {
+    name: "CometAdminRte",
+    slot: "editor",
+    overridesResolver({ ownerState }: { ownerState: OwnerState }, styles) {
+        return [styles.editor, ownerState.disabled && styles.disabled];
+    },
+})<{ ownerState: OwnerState }>(
+    ({ ownerState, theme }) => css`
+        & .public-DraftEditor-content {
+            min-height: 240px;
```

Comment:
This is different, than the previous implementation: `${props.minHeight === undefined ? 240 : props.minHeight}px`